src/screens/Wallets/NumberPadButtons.tsx

```diff
@@ -68,9 +68,9 @@ const NumberPadButtons = ({
 						onPress={onChangeUnit}>
 						<SwitchIcon color={color} width={16.44} height={13.22} />
 						<Caption13Up color={color} style={styles.middleButtonText}>
-							{nextUnit === 'BTC' && 'BTC'}
-							{nextUnit === 'satoshi' && 'sats'}
-							{nextUnit === 'fiat' && fiatTicker}
+							{nextUnit === 'BTC' && fiatTicker}
+							{nextUnit === 'satoshi' && 'BTC'}
+							{nextUnit === 'fiat' && 'sats'}
```

Comment:
In this component we can use `unit` directly and get rid of `nextUnit` for a small cleanup.

```suggestion
							{unit === EUnit.BTC && 'BTC'}
							{unit === EUnit.satoshi && 'sats'}
							{unit === EUnit.fiat && fiatTicker}
```